internal/api/v1/docs/application.go

```diff
@@ -0,0 +1,242 @@
+package docs
+
+//go:generate swagger generate spec
+
+import "github.com/epinio/epinio/pkg/api/core/v1/models"
+
+// swagger:route GET /applications application AllApps
+// Return list of applications in all namespaces.
+// responses:
+//   200: AppsResponse
+
+// swagger:parameters AllApps
+type AllAppsParam struct{}
+
+// response: See Apps.
+
+// swagger:route GET /namespaces/{namespace}/applications application Apps
+// Return list of applications in the `namespace`.
+// responses:
+//   200: AppsResponse
+
+// swagger:parameters Apps
+type AppsParam struct {
+	Namespace string
+}
+
+// swagger:response AppsResponse
+type AppsResponse struct {
+	// in: body
+	Body models.AppList
+}
+
+// swagger:route POST /namespaces/{namespace}/applications application AppCreate
+// Create the posted new application in the `namespace`.
+// responses:
+//   200: AppCreateResponse
+
+// swagger:parameters AppCreate
+type AppCreateParam struct {
+	Namespace string
+	// in: body
+	Service models.ApplicationCreateRequest
+}
+
+// swagger:response AppCreateResponse
+type AppCreateResponse struct {
+	// in: body
+	Body models.Response
+}
+
+// swagger:route GET /namespaces/{namespace}/applications/{app} application AppShow
+// Return details of the named `application` in the `namespace`.
+// responses:
+//   200: AppShowResponse
+
+// swagger:parameters AppShow
+type AppShowParam struct {
+	Namespace string
+	App       string
+}
+
+// swagger:response AppShowResponse
+type AppShowResponse struct {
+	// in: body
+	Body models.App
+}
+
+// swagger:route GET /namespaces/{namespace}/applications/{app}/logs application AppLogs
+// Return logs of the named `application` in the `namespace`.
+// responses:
+//   200: AppLogsResponse
+
+// swagger:parameters AppLogs
+type AppLogsParam struct {
+	Namespace string
+	App       string
+}
+
+// swagger:response AppLogsResponse
+type AppLogsResponse struct {
+	// in: body
+}
+
+// swagger:route GET /namespaces/{namespace}/staging/{stage_id}/logs application StagingLogs
+// Return logs of the named `stage_id` in the `namespace`.
+// responses:
+//   200: StagingLogsResponse
+
+// swagger:parameters StagingLogs
+type StagingLogsParam struct {
+	Namespace string
+	StageId   string
+}
+
+// swagger:response StagingLogsResponse
+type StagingLogsResponse struct {
+	// in: body
```

Comment:
I haven't found what to add for the `LogsResponse`... Any hints?

Reply:
See above.